test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -33,11 +33,15 @@ def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
         hide=True,
     )
 
-    ec2_connection.run(
+    test_output = ec2_connection.run(
         f"nvidia-docker exec --user root smdebug-gpu /bin/bash -c '{test_script} {framework}'",
-        hide=True,
+        hide=True, warn=True, timeout=5000
```

Comment:
nit: this timeout won't ever be reached in PR context, due to the global timeout of 4860 seconds. It's fine to keep it if you think it makes sense to have this as a high timeout for other instance types.

Reply:
Sure, we can drop this timeout if it makes no sense.